docs/site/pages/assets/code-editor.mdx

```diff
@@ -0,0 +1,90 @@
+# @devtools-ui/code-editor
```

Comment:
LGTM, nextjs routing makes `/code-editor` available for routing, but for this document to be listed on the side navigation bar it needs to get listed on these routes around [here](https://github.com/player-ui/devtools-assets/blob/main/docs/site/config/navigation.ts#L75). This is something the updated `gen: asset` can take care of for you in future assets.